ci/test.sh

```diff
@@ -53,6 +53,10 @@ echoinfo "Test complete"
 
 echoinfo "Uploading test results"
 
+# Determine CODACY_REPORTER_VERSION once to avoid hitting the "get latest" API too frequently
+# in the get.sh commands below. Note: the sed command was copied from get.sh
+export CODACY_REPORTER_VERSION="$(curl https://api.bintray.com/packages/codacy/Binaries/codacy-coverage-reporter/versions/_latest | sed -e 's/.*name[^0-9]*\([0-9]\{1,\}[.][0-9]\{1,\}[.][0-9]\{1,\}\).*/\1/' )"
```

Comment:
Could we just set this to `latest`?
https://github.com/codacy/codacy-coverage-reporter/commit/10a9742490a7558e43839b253285ae2f64e7e435